tests/spec/core/biblio-spec.js

```diff
@@ -102,6 +107,18 @@ describe("W3C — Bibliographic References", () => {
     expect(ref.textContent).toMatch(/Publisher Here\.\s/);
   });
 
+  it("displays the reference for APA citation", async() => {
+    // Make sure the reference is added.
+    let ref = doc.querySelector("#bib-TestRef4 + dd");
+    expect(ref).toBeTruthy();
+    // This prevents Jasmine from taking down the whole test suite if SpecRef is down.
+    if (!specRefOk) {
```

Comment:
We shouldn't need this here, as this is a local reference.

Reply:
Alright, so no assertions needed?

Reply:
just the `if (!specRefOk) { ... ` can go away, as it doesn't serve any useful purpose here. `specRefOk` is for checking if we can talk to the Specref server - but here, we don't talk to the server.